CHANGELOG.md

```diff
@@ -6,26 +6,62 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v17.0.0] 2023-08-09
+
+### MIGRATION notes
+
+- This release updates the `hashicorp/aws` provider required by Cumulus to `~> 5.0`
+  which in turn requires updates to all modules deployed with Core in the same stack
+  to use a compatible provider version.
+- This update is *not* compatible with prior stack states - Terraform will not
+  allow redeployment of a prior version of Cumulus using an older version of
+  the provider.  Please be sure to validate the install changeset is what you
+  expect prior to upgrading to this version.
+- Migration forward to v17 from v16+ is strongly
+  recommended, as the changeset from the v16.x.y baselines is relatively minimal
+  and has been tested.
```

Comment:
Maybe we could say "Upgrading to v16.x before v17.x is recommended..." Would that be clearer?

Reply:
The intent here is to caution against going from, say v11->v16 as rollback is problematic.